core/store/src/test/java/org/locationtech/geowave/core/store/api/DataStoreRemoveTest.java

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright (c) 2013-2019 Contributors to the Eclipse Foundation
+ *
+ * <p> See the NOTICE file distributed with this work for additional information regarding copyright
+ * ownership. All rights reserved. This program and the accompanying materials are made available
+ * under the terms of the Apache License, Version 2.0 which accompanies this distribution and is
+ * available at http://www.apache.org/licenses/LICENSE-2.0.txt
+ */
+package org.locationtech.geowave.core.store.api;
+
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.locationtech.geowave.core.store.BaseDataStoreOptions;
+import org.locationtech.geowave.core.store.DataStoreOptions;
+import org.locationtech.geowave.core.store.StoreFactoryFamilySpi;
+import org.locationtech.geowave.core.store.StoreFactoryOptions;
+import org.locationtech.geowave.core.store.adapter.MockComponents;
+import org.locationtech.geowave.core.store.index.IndexImpl;
+import org.locationtech.geowave.core.store.memory.MemoryStoreFactoryFamily;
+
+public class DataStoreRemoveTest {
+  private static int counter = 0;
+  private static final String MOCK_DATA_TYPE_1 = "Some Data Type";
+  private static final String MOCK_DATA_TYPE_2 = "Another Data Type";
+
+  private DataStore dataStore;
+
+  @Before
+  public void createStore() {
+    dataStore = DataStoreFactory.createDataStore(new MemoryRequiredNameSpaceOptions(counter + ""));
```

Comment:
now that it is properly deleting everything, can we clean this up and remove this and associated inner class

Reply:
Fixed in commit.